src/translations/nbTranslations.json

```
@@ -797,6 +797,8 @@
       "preferred_last_name": "Foretrukket etternavn",
       "preferred_name": "Forfatternavn",
       "profile_picture": "Profilbilde",
+      "upload_is_not_mandatory": "<0>Det er frivillig å laste opp et bilde.</0><0>Du kan når som helst trekke tilbake samtykket ved å slette bildet.</0>",
+      "upload_description": "<0>Ved å laste opp et bilde samtykker du til at bilde legges inn som profilbilde på din offentlige profil i Nasjonalt vitenarkiv.</0><0>Profilbilde er offentlig synlig for alle som benytter Nasjonalt vitenarkiv.</0>",
```

(ikke 100% sikker på gramatikken her)
```suggestion
      "upload_description": "<0>Ved å laste opp et bilde samtykker du til at bilde legges inn som profilbilde på din offentlige profil i Nasjonalt vitenarkiv.</0><0>Profilbildet er offentlig synlig for alle som benytter Nasjonalt vitenarkiv.</0>",
```

---

Det er riktig som du sier